pom.xml

```diff
@@ -567,13 +567,13 @@
 			<dependency>
 				<groupId>org.testcontainers</groupId>
 				<artifactId>mysql</artifactId>
-				<version>1.21.1</version>
+				<version>1.21.2</version>
```

Comment:
is this change supposed to be part of the pull request that changes patient and person from hbm to annotations?